watertap/unit_models/tests/test_gac.py

```diff
@@ -418,7 +419,7 @@ def test_robust_costing_gravity(self, gac_frame_robust):
 
         mr_grav.fs.unit.costing = UnitModelCostingBlock(
             flowsheet_costing_block=mr_grav.fs.costing,
-            costing_method_arguments={"contactor_type": "gravity"},
+            costing_method_arguments={"contactor_type": ContactorType.gravity},
```

Comment:
The whole point of the `StrEnum` is you don't have to import these enums everywhere to use them. As an example I like the prior code better.

Reply:
This is fine to change back, I made it just to have my tests passing as I was playing around with changing names of the `StrEnum` options.